packages/localstorage/localstorage.js

```diff
@@ -21,7 +21,33 @@ try {
 } catch (ignored) {}
 
 if (key === retrieved) {
-  Meteor._localStorage = storage;
+  if (Meteor.isServer) {
+    Meteor._localStorage = storage;
+  } else {
+    // IE11 doesn't handle properly attempts to change methods of the
+    // window.localStorage, attempts to do so will result in the complete
+    // break of the localStorage system for the domain in which it is
+    // done - until the user clean the browser/domain cache.
+    //
+    // Therefore, in the web, we don't set Meteor._localStorage to be a
+    // reference to window.localStorage . Instead, we set proxy methods.
+    //
+    // This will allow package developers that will find a need to change
+    // the behavior of Meteor._localStorage methods to do so without breaking
+    // the localStorage system on IE11. (e.g. meteorhacks:fast-render)
+
+    Meteor._localStorage = {
+      getItem: function (key) {
+        return window.localStorage.getItem(key);
+      },
+      setItem: function (key, value) {
+        window.localStorage.setItem(key, value);
+      },
+      removeItem: function (key) {
+        window.localStorage.removeItem(key);
+      }
+    };
```

Comment:
This might be a bit of an edge case, but this change could break backwards compatibility. It looks like we're replacing the full `global.localStorage` object that is stored in `Meteor._localStorage` with a proxied version that doesn't include all `global.localStorage` properties. For example, someone could be calling `Meteor._localStorage.key(0)` (see [`Storage.key()`](https://developer.mozilla.org/en-US/docs/Web/API/Storage/key)); this would break using this new proxied `Meteor._localStorage` version. Then again we aren't replicating `key()` in the [manually built `Meteor._localStorage` object](https://github.com/theosp/meteor/blob/0a616060125eeacafc600220bd3a4fd8d88d48bd/packages/localstorage/localstorage.js#L61) either, so maybe this is okay? Food for thought anyways.